osu.Game/Graphics/UserInterface/Counter.cs

```diff
@@ -0,0 +1,110 @@
+﻿//Copyright (c) 2007-2016 ppy Pty Ltd <contact@ppy.sh>.
+//Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using OpenTK;
+using OpenTK.Graphics;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+
+namespace osu.Game.Graphics.UserInterface
+{
+    public abstract class Count : AutoSizeContainer
+    {
+        private Sprite buttonSprite;
+        private Sprite glowSprite;
+        private SpriteText keySpriteText;
+        private SpriteText countSpriteText;
+
+        public override string Name { get; }
+        public KeyCounter ParentCounter { get; set; }
+        public int Counts { get; private set; }
+
+        private bool isLit;
+        public bool IsLit
+        {
+            get { return isLit; }
+            protected set
+            {
+                if (isLit != value)
+                {
+                    isLit = value;
+                    UpdateGlowSprite();
+                    if (value && ParentCounter.IsCounting)
+                        IncreaseCount();
+                }
+            }
+        }
+
+        public Color4 KeyDownTextColor { get; set; } = Color4.DarkGray;
+        public Color4 KeyUpTextColor { get; set; } = Color4.White;
+
+        protected Count(string name)
+        {
+            Name = name;
+        }
+
+        public override void Load()
+        {
+            base.Load();
+            Children = new Drawable[]
+            {
+                buttonSprite = new Sprite
+                {
+                    Texture = Game.Textures.Get(@"KeyCounter/key-up"),
+                    Anchor = Anchor.Centre,
+                    Origin = Anchor.Centre,
+                    Children = new Drawable[]
+                    {
+                        glowSprite = new Sprite

```

Comment:
You probably shouldn't put a Sprite in a Sprite. This used to be an enforced condition (Add was not accessible for non-containers) and will likely be added back at some point.

Just put it at the same level as buttonSprite.